src/system/BUILD.gn

```diff
@@ -43,6 +43,8 @@ declare_args() {
   #
   # see https://clang.llvm.org/docs/ThreadSafetyAnalysis.html
   chip_enable_thread_safety_checks = is_clang
+
+  chip_external_sdk_target = ""
```

Comment:
please add documentation on how this is indended to be used.

Why don't we default this to the `${efr32_sdk_build_root}:efr32_sdk"` or other values depending on the platform and then let others override this? This seems to be more consistent than adding a completely separate flag.

Reply:
I would propose a `chip_platform_sdk_target` path that gets initialized like the existing deps, but allows overriding.